File: old/PHX_submit.py
#!/usr/bin/env python
#SBATCH -J PHX_Submission	# The job name

import sys
import subprocess

for s in range(0, int(sys.argv[1])):
	subprocess.call(['sbatch', '-J', 'RunPD_' + str(s), '-o', 'RunPD_' + str(s), 'PHX_run.py', str(s), str(sys.argv[2])])

File: misc/PHX_submit.pl
#!/usr/bin/perl

my @problem_0 = ("0_1", "3_4");
my @problem_1 = ("2_4", "3_1");

for (my $p = 0; $p < scalar(@problem_0); $p++){
	system("sbatch -J $problem_0[$p]_rerun -o $problem_0[$p]_rerun PHX_run.pl $problem_0[$p]");
}

for (my $p = 0; $p < scalar(@problem_1); $p++){
	system("sbatch -J $problem_1[$p]_rerun -o $problem_1[$p]_rerun PHX_run.problem_1 $problem_0[$p]");
}